src/System-Support/SystemNavigation.class.st

```diff
@@ -236,19 +236,28 @@ SystemNavigation >> allSentMessages [
 ]
 
 { #category : #query }
-SystemNavigation >> allUnimplementedCalls [
+SystemNavigation >> allSentNotImplementedSelectors [
 
 	"Answer all methods where a message is sent but the selector is not implemented anywhere in the system."
 
-	^ self allMethods select: [ :meth | 
+	^ self allMethods select: [ :meth |
 		  | ignored |
-		  ignored := (meth pragmaAt: #ignoreUnimplementedCalls:)
+		  ignored := (meth pragmaAt: #ignoreNotImplementedSelectors:)
```

Comment:
As we use this checking the pragma part now a second time could we not delegate this question to the method itself and extract it into an own "accessing" method like #allIgnoredNotImplementedSelectors?

```Smalltalk
allIgnoredNotImplementedSelectors
      ^ (self pragmaAt: #ignoreNotImplementedSelectors:)
                        ifNotNil: [ :pragma | pragma argumentAt: 1 ]
			ifNil: [ #(  ) ].
```

Reply:
this would then make the code more easy to read as you just reject "allIgnoredNotImplementedSelectors"

Reply:
DONE